### PR TITLE
Create bottom navbar

### LINKED ISSUE #26: Create bottom navbar
### User Story
As a user and as a developer I need a navigation for this app. It should be at the bottom of the app. 

### To do
-Create a navbar with: 
- [x] active/inactive Buttons
- [x] design 

### Criteria
- [ ] Use design prototype
- [ ] Use storybook to build the components

### Material | Images
Design:
https://xd.adobe.com/view/7f86dc69-703e-48db-70d4-8d3d07276926-09b7/

design examples:
image

### PR DESCRIPTION
**Pull request by meloneminze**
I created a bottom navbar(ul) with two list items. I think there is a better way to active the polygon. Maybe you have a solution for me how the refactoring the active code. 

image

close #26

### CODE REVIEW

File: client/src/stories/NavBottom.stories.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  return (
+    <NavBottom />
+    //       links={[
+    //         { label: 'Children', Icon: Child },
+    //         { label: 'Household', Icon: Home },
+    //       ]}
+    //       value={active}
+    //       onItemClick={(page) => setActive(page)}
+    //     />
```
Comment (meloneminze) [inline client/src/stories/NavBottom.stories.js:22]: There i have to delete the outlined code.

File: client/src/components/NavBottom.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    <NavContainer>
+      <NavItemLeft active={location.pathname === '/'}>
+        <NavLink to="/">
+          <SVG
+            xmlns="http://www.w3.org/2000/svg"
+            width="24"
+            height="24"
+            fill="#4D4D4D"
+            viewBox="0 0 24 24"
+            active={location.pathname === '/'}
```
Comment (marwinburesch) [inline client/src/components/NavBottom.js:63]: I don't think this does anything

File: client/src/components/NavBottom.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { Link, useLocation } from 'react-router-dom';
+
+const NavContainer = styled.ul`
+  display: flex;
+  // justify-content: center;
+  // align-items: center;
```
Comment (derrabus) [inline client/src/components/NavBottom.js:8]:
```suggestion
```

Comment (meloneminze) [reply client/src/components/NavBottom.js:8]: Thanks! I missed that.

File: client/src/components/NavBottom.js (hunk 2 above)
Comment (meloneminze) [reply client/src/components/NavBottom.js:63]: You are right.

**Pull request merged**